### ISSUE TITLE
Issue duplication

### ISSUE THREAD

**Issue by rodrigoGA** [jenkinsci/git-changelog-plugin#65]
I am using jenkins, with the following command:

```
   def changelogContext = gitChangelog returnType: 'CONTEXT',
                    from: [type: 'REF', value: "${LAST_TAG}"],
                    to: [type: 'REF', value: "${NEXT_TAG}"],
                    jira: [issuePattern: 'MYPROJ_CS-([0-9]+)']


   changelogContext.issues.each { issue ->
                    print("----------------------------------")
                    print("issue.name  ${issue.name}")
                    print("infoTicket.ticket ${infoTicket.ticket}")

                    for (def commit : issue.commits){ 
                        print("hash ${commit.hash}")
                    }
   }

```

and in the console output I can see

```
----------------------------------
issue.name Jira
infoTicket.ticket EASYAP_CS-68350
hash 1f15c298ecbb8e6
hash 3c20d05ab137002
----------------------------------
issue.name Redmine
infoTicket.ticket 
hash 1f15c298ecbb8e6
```

For some reason it is detecting a redmine issue, when I don't have it configured.

**Comment by tomasbjerre** [comment]
A workaround might be to change it like:

```
   def changelogContext = gitChangelog returnType: 'CONTEXT',
                    from: [type: 'REF', value: "${LAST_TAG}"],
                    to: [type: 'REF', value: "${NEXT_TAG}"],
                    jira: [issuePattern: 'MYPROJ_CS-([0-9]+)'],
                    redmine: [issuePattern: 'thiswillnevermatch'],
                    github: [issuePattern: 'thiswillnevermatch'],
                    gitlab: [issuePattern: 'thiswillnevermatch']
```

**Comment by tomasbjerre** [comment]
Releasing a fix for this now in `3.15`. Open issue again if not working.
